OpenRA.Mods.Common/Scripting/Properties/ProductionProperties.cs

```diff
@@ -90,7 +90,7 @@ public class ProductionQueueProperties : ScriptActorProperties, Requires<Product
 		public ProductionQueueProperties(ScriptContext context, Actor self)
 			: base(context, self)
 		{
-			queues = self.TraitsImplementing<ProductionQueue>().Where(q => q.Enabled).ToArray();
+			queues = self.TraitsImplementing<ProductionQueue>().Where(q => q.UsableByFaction).ToArray();
```

Comment:
`UsableByFaction` can change at runtime, so this won't work.  It looks like this was broken before you got here, but lets fix it now.